### PR TITLE
Fixing ethereum bridge

### PR DESCRIPTION
**Pull request by HackFisher**

### CODE REVIEW

**Comment by HackFisher** [comment]
@hammeWang @AurevoirXavier Ready for Review.

Because develop branch should not be broken, so I commented out the ```sr-eth-primitives``` under ```ethereum-bridge```.

Can uncomment them for testing purpose after being merged.

**Comment by hammeWang** [comment]
why remove the keccak mod? https://github.com/darwinia-network/darwinia/pull/118/files#diff-931ba1cf79fbb72e7081a758063920e0

**Comment by HackFisher** [comment]
> why remove the keccak mod? https://github.com/darwinia-network/darwinia/pull/118/files#diff-931ba1cf79fbb72e7081a758063920e0

Not need anymore, there was a unsafe code and using std.

https://github.com/darwinia-network/darwinia/pull/118/files#diff-a52d5efaba7b2addecda30eaaa4e5005L303

Directly using ```keccak_hash::keccak_256``` now.

File: core/sr-eth-primitives/Cargo.toml [hunk 1]
```diff
@@ -1,61 +1,51 @@
...
+rlp = { package = "rlp", git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+rlp_derive = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false }
+primitive-types = { git = "https://github.com/darwinia-network/parity-common.git", default-feature = false, features = ["codec", "rlp"] }
+ethereum-types = { git = "https://github.com/darwinia-network/parity-common.git", default-feature = false, features = ["serialize"] }
+keccak-hash = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+impl-codec = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+fixed-hash = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+impl-rlp = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+ethbloom = { package = 'ethbloom', git ="https://github.com/darwinia-network/parity-common.git", default-features = false, features = ["serialize"]}
```
Comment (AurevoirXavier) [inline core/sr-eth-primitives/Cargo.toml]: package field can be removed

```rust
ethbloom = { package = 'ethbloom', git ="https://github.com/darwinia-network/parity-common.git", default-features = false, features = ["serialize"]}
```

File: core/sr-eth-primitives/Cargo.toml [hunk 2]
```diff
@@ -1,61 +1,51 @@
...
-keccak-hash = "0.4.0"
-impl-codec = { version = "0.4", default-features = false, optional = true }
-fixed-hash = { version = "0.4", default-features = false}
-impl-rlp = { version = "0.2", default-features = false, optional = true }
-hbloom = { package = 'ethbloom', git ="https://github.com/hammeWang/ethbloom.git", default-features = false, features = ["serialize"]}
-crunchy = { version = "0.2.2", default-features = false, features = ["limit_256"] }
+#substrate-primitives = {git = "https://github.com/darwinia-network/substrate.git", branch = "darwinia-develop", default-features = false }
+rlp = { package = "rlp", git = "https://github.com/darwinia-network/parity-common.git", default-features = false}
+rlp_derive = { git = "https://github.com/darwinia-network/parity-common.git", default-features = false }
+primitive-types = { git = "https://github.com/darwinia-network/parity-common.git", default-feature = false, features = ["codec", "rlp"] }
```
Comment (AurevoirXavier) [inline core/sr-eth-primitives/Cargo.toml]: missing 's': `default-features`

File: core/sr-eth-primitives/Cargo.toml [hunk 3]
```diff
@@ -1,61 +1,51 @@
...
+ethereum-types = { git = "https://github.com/darwinia-network/parity-common.git", default-feature = false, features = ["serialize"] }
```
Comment (AurevoirXavier) [inline core/sr-eth-primitives/Cargo.toml]: missing 's': `default-features`

**Pull request merged**